plugins/modules/grpc_nw_config.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: grpc_nw_config
+version_added: ""
+author:
+    - "Gomathi Selvi S (@GomathiselviS)"
+short_description: Fetch configuration/state data from gRPC enabled target hosts.
+description:
+    - gRPC is a high performance, open-source universal RPC framework.
+    - This module allows the user to append configs to an existing configuration in a gRPC
+      enabled devices.
+options:
+  config:
+    description:
+      - This option specifies the string which acts as a filter to restrict the portions of
+        the data to be retrieved from the target host device. If this option is not specified the entire
+        configuration or state data is returned in response provided it is supported by target host.
+  path:
```

Comment:
I think it is better to have option as `config` which accepts the input config in json format instead of file `path` as user can read from file using file lookup plugin or provide values from `host_vars`

Reply:
Done